tests/integration/django/test_workers.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Integration tests for Django charm."""
+import asyncio
+import logging
+import time
+import typing
+
+import pytest
+import requests
+from juju.application import Application
+from juju.model import Model
+from juju.utils import block_until
+from pytest_operator.plugin import OpsTest
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.mark.parametrize(
```

Comment:
Do you think these could be more unit tests of the config validation?

Reply:
Also, should we send a request to an app and make sure we get a response?

Reply:
Moved the config tests to unit and add a integration test that sends a request to the app